docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
+
+### Get started with Lando
+
+The following examples are based on the [PHP template](https://github.com/platformsh-templates/php).
+
+Several stacks are available on Lando (LAMP, LEMP, MEAN).
+If you use a specific stack, adapt the steps below accordingly.
+See the [Lando documentation for all recipes and stacks available](https://docs.lando.dev/).
+
+Lando requires a `.lando.yml` file. The `init` command can generate that file automatically.
+
+The full output of the commands below have been redacted for brevity.
+
+**TODO: fix the Lando init step for both paths + remove the need for the prompt (just add a note) + add stack + add cloning of the PHP base project**
+
+{{< codetabs >}}
+
+---
+title=On a new project
+file=none
+highlight=false
+---
+
+Run `lando init --recipe platformsh --source platformsh` and follow the instructions provided by the interactive prompt.
+
+``` bash
 $ lando init --source platformsh
+
+? Select a Platform.sh account <email address of your account>
+? Which project? <project name>
 ```
 
-That will offer an interactive dialog to select the Platform.sh project to download, and then setup basic Lando configuration.
+<--->
+
+---
+title=On an existing project
+file=none
+highlight=false
+---
+
+Open a shell and access the directory where your project is located.
 
-If you already have your project checked out locally, change into the project directory and run:
+Enter `lando init --recipe platformsh --source cwd` and follow the instructions provided by the interactive prompt.
 
-```bash
-$ lando init --source cwd --recipe platformsh
+``` bash
+$ lando init --source platformsh --source cwd --webroot .
+? From where should we get your app's codebase? current working directory
+? What recipe do you want to use? platformsh
+? Select a Platform.sh account <email address of your account>
+? Which project? <project name>
 ```
 
-That will use your existing git clone and add the appropriate Lando configuration.
+{{< /codetabs >}}
 
-Either way, start the local environment by running:
+Once the command ran successfully, you get the following:
 
-```bash
+``` bash
+   _  __                       _
+  / |/ /__ _    __  _    _____( )_______
+ /    / _ \ |/|/ / | |/|/ / -_)// __/ -_)
+/_/|_/\___/__,__/  |__,__/\__/ /_/  \__/
+
+  _________  ____  __ _______  _______  _      ______________ __  ___________  ______
+ / ___/ __ \/ __ \/ //_/  _/ |/ / ___/ | | /| / /  _/_  __/ // / / __/  _/ _ \/ __/ /
+/ /__/ /_/ / /_/ / ,< _/ //    / (_ /  | |/ |/ // /  / / / _  / / _/_/ // , _/ _//_/
+\___/\____/\____/_/|_/___/_/|_/\___/   |__/|__/___/ /_/ /_//_/ /_/ /___/_/|_/___(_)
+
+Your app has been initialized!
+
+Go to the directory where your app was initialized and run lando start to get rolling.
+Check the LOCATION printed below if you are unsure where to go.
+
+Oh... and here are some vitals:
+
+ NAME      <project name>
```

Comment:
This isn't a variable and so shouldn't really be formatted this way. To approach this in a tutorial way, it should have example values that are clearly examples.

But again, not sure it's necessary to have this block at all.